services/utils/conditional-headers/pom.xml

```diff
@@ -33,6 +33,16 @@
             <groupId>org.eclipse.ditto</groupId>
             <artifactId>ditto-signals-commands-base</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.ditto</groupId>
+            <artifactId>ditto-signals-commands-things</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Duplicated `<dependency>` block. Please remove one of them.

Reply:
Done